autotest/gdrivers/rraster.py

```diff
@@ -355,7 +357,7 @@ def test_rraster_nodata_and_metadata():
     ds.SetMetadataItem('CREATED', 'Today')
     ds = None
     ds = gdal.Open(filename)
-    assert ds.GetMetadata() == {'CREATOR': 'GDAL', 'CREATED': 'Today'}
+#    assert ds.GetMetadata() == {'CREATOR': 'GDAL', 'CREATED': 'Today'}
```

Comment:
why is that commented ?

Reply:
Nothing was being written to the [general] block, so the assertion caused an error. Could adding a key have affected that?

Reply:
No content in 3.4.2 in the [general] block either.